lib/process_settings/file_monitor.rb

```diff
@@ -24,6 +26,31 @@ def initialize(file_path, logger:, environment: nil)
       start_internal(enable_listen_thread?(environment))
     end
 
+    def start_watchdog_thread(file_path = :missing)
+      if file_path == :missing
+        ActiveSupport::Deprecation.warn("ProcessEnvSetup.start_watchdog_thread with no arguments is deprecated")
+        file_path = ProcessSettings::Monitor.file_path
+      end
```

Comment:
No need to still support this deprecation stuff since you've moved the code--it's not possible for anyone to have been calling it here, the old way.